### ISSUE TITLE
Snackbar Accessibility Audit

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#9495]
## Description
Since we do not have confirmation screens in the app, snackbars / toast messages can be an excellent way to notify users (primarily those who use a screenreader or have a cognitive disability) that a change has occurred (a message was sent or failed to send, an image/file was upload or failed, etc.).

### Acceptance Criteria
- [ ] Audit the app and put together a list of where we use snackbars currently.
- [ ] Determine if there are any places that we do not use snackbars and should.

**Comment by brea11y** [comment]
@dumathane – Would it be possible to get a list of all of the snackbars used in the app today? Similarly to the way that we pulled buttons and links, alerts, etc. If so, who would be the best person to loop in to grab them for this audit?
